src/task_impl/std/mod.rs

```diff
@@ -8,21 +8,17 @@ use std::ptr;
 use std::sync::{Arc, Mutex, Condvar, Once, ONCE_INIT};
 use std::sync::atomic::{AtomicUsize, Ordering};
 
-use {Future, Stream, Sink, Poll, Async, StartSend, AsyncSink};
+use {Future, Stream, Sink, Async, AsyncSink};
 use super::core;
 use super::{BorrowedTask, NotifyHandle, Spawn, spawn, Notify, UnsafeNotify};
+pub use super::core::{BorrowedUnpark, TaskUnpark};
```

Comment:
Now that there's no need for unpark events this can (in the future of course if you'd prefer) be cleaned up a lot and not have this sd/core business. These types can move from the `core` module directly into the parent module here to directly make up `Task` and remove a few layers of indirection in the meantime.